### ISSUE TITLE
Tasks 1.1.15 - 1.1.17 skipped

### ISSUE THREAD

**Issue by flwitten** [ansible-lockdown/RHEL8-CIS#132]
**Describe the Issue**
In task 1.1.15 - 1.1.17 it is supposed to set mount options for /dev/shm.
As /dev/shm is not listed in ansible_mounts this task will never get executed.
It's not listed even if /etc/fstab contains following line:
`tmpfs /dev/shm tmpfs defaults,noexec,nodev,nosuid 0 0`

**Expected Behavior**
Task will execute

**Actual Behavior**
Task is skipped

**Environment :**
 - Ansible Version: 2.9.24
 - Host Python Version: 3.6.8
 - Ansible Server Python Version: 3.6.8
 - Additional Details: Oracle Linux 8.4

**Possible Solution**
I'm using following as a workaround:
```
mount:
  path: /dev/shm
  src: tmpfs
  fstype: tmpfs
  state: present
  opts: defaults,noexec,nodev,nosuid
```

**Comment by georgenalen** [comment]
@flwitten 
 Thanks for raising the issue (and for being the first to raise an issue using the issue templates). I have added a pre-check for the /dev/shm mount and then make the changes needed. We will be pushing a release in the next day or so, I will close this ticket once it hits the main branch. Thanks again for raising the issue, they help a lot in making this improve for the better. 

George

**Comment by georgenalen** [comment]
This issue as been resolved with the release of 1.40
